sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeActionsOperationDetail.java

```diff
@@ -15,10 +15,10 @@
 public final class AnalyzeActionsOperationDetail {
     private String operationId;
     private String displayName;
-    private int actionsFailed;
-    private int actionsInProgress;
-    private int actionsSucceeded;
-    private int actionsInTotal;
+    private int failedCount;
```

Comment:
I liked the older names 🙆 
But if thats what the team decided 👍

Reply:
Jonathan and Srikanta think these names are better represent the numbers